server/routes/api/users.js

```diff
@@ -27,9 +27,7 @@ router.post("/register", (req, res) => {
 
   User.findOne({ email: req.body.email }).then(user => {
     if (user) {
-      return res
-        .status(400)
-        .json({ emailAlreadyExists: "Email already exists" });
+      return res.json({ emailDoesNotExist: "Email already exists" });
```

Comment:
I understand that you switched your validation where `false` was once `true`, but this line doesn't make any sense. I would rename the key to `duplicateEmail`. It will make less sense in the cases of `inputValidation.duplicateEmail = false`, but it is easier to understand when you see how all other `inputValidation` values are `true` when valid and `false` when invalid.